Build/samples/V1/PublishBuildEventClient/publish_build_tool_event_stream.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was automatically generated - do not edit!
+ */
+
+require_once __DIR__ . '/../../../vendor/autoload.php';
+
+// [START buildeventservice_v1_generated_PublishBuildEvent_PublishBuildToolEventStream_sync]
```

Comment:
Need to investigate the product name (would think this should be `build`).

Reply:
I see what happened, when we updated to owlbot we used the following source: https://github.com/googleapis/google-cloud-php/blob/2cc2210d3839689943416a223d29928dd93cf1ac/Build/.OwlBot.yaml#L2

It should be:

`source: /google/devtools/cloudbuild/v1/.*-php/(.*)`